spec.html

```diff
@@ -564,9 +564,19 @@ <h1>Runtime Semantics: ClassFieldDefinitionEvaluation</h1>
     1. Else,
       1. Let _initializer_ be ~empty~.
       1. Let _isAnonymousFunctionDeclaration_ be *false*.
+    1. If _key_ is a Private Name,
+      1. Let _enumerable_ be *false*.
```

Comment:
Private names aren't properties, so I wouldn't expect them to have descriptors at all.

Reply:
Private methods descriptor have a `false/false/false` descriptor, so I copied it. Also, you could make a private element non-writable using a decorator.

Reply:
You can make a `value` private field become a `get/set` private field.
And yes, `configurable/enumerable` don't make sense, so that are always set to false (iirc, decorators throw if you return a private element descriptor which is enumerable or configurable).

Reply:
Thanks for correctly answering these questions, @nicolo-ribaudo . Does anyone have an idea about what might be helpful to include in a note in the specification, to make it more clear?

Reply:
Maybe adding assertions that [[Configurable]] and [[enumerable]] are false when private descriptors are used is enough?

Reply:
I'm not sure that would be enough; @ljharb didn't seem to understand that private uses descriptors at all.